camel-spring-boot-upgrade-recipes/src/main/resources/META-INF/rewrite/latest.yaml

```diff
@@ -21,54 +21,4 @@ displayName: Migrate to Apache Camel Spring Boot @camel-version@
 description: >- 
   Migrate applications to Apache Camel Spring Boot @camel-version@ and Spring Boot @spring-boot-version@
 recipeList:
-  - org.apache.camel.upgrade.camel412.CamelMigrationRecipe
-  - org.apache.camel.upgrade.camel411.CamelMigrationRecipe
-  - org.apache.camel.upgrade.camel410.CamelMigrationRecipe
-  - org.apache.camel.upgrade.camel49.CamelMigrationRecipe
-  - org.apache.camel.upgrade.camel47.CamelMigrationRecipe
-  - org.apache.camel.upgrade.camel46.CamelMigrationRecipe
-  - org.apache.camel.upgrade.camel45.CamelMigrationRecipe
-  - org.apache.camel.upgrade.camel44.CamelMigrationRecipe
-  - org.apache.camel.upgrade.camel40.CamelMigrationRecipe
-  - org.apache.camel.upgrade.UpgradeToJava17
-  - org.apache.camel.upgrade.JavaVersion17
-  - org.apache.camel.upgrade.UpdateCamelSpringBootPropertiesAndYamlKeys
-  - org.openrewrite.maven.UpgradeDependencyVersion:
-      groupId: '*camel*'
-      artifactId: 'camel-spring-boot-bom'
-      newVersion: @camel-spring-boot-version@
-  - org.openrewrite.maven.UpgradeDependencyVersion:
-      groupId: 'org.apache.camel.springboot'
-      artifactId: 'spring-boot'
-      newVersion: @camel-spring-boot-version@
-  - org.openrewrite.maven.UpgradeDependencyVersion:
-      groupId: 'org.apache.camel.springboot'
-      artifactId: 'camel-spring-boot-dependencies'
-      newVersion: @camel-spring-boot-version@
-  - org.openrewrite.maven.UpgradeDependencyVersion:
-      groupId: 'org.apache.camel.springboot'
-      artifactId: 'camel-spring-boot-dependencies'
-      newVersion: @camel-spring-boot-version@
-  - org.openrewrite.maven.UpgradeDependencyVersion:
-      groupId: org.springframework.boot
-      artifactId: "*"
-      newVersion: @spring-boot-version@
-      overrideManagedVersion: false
-  - org.openrewrite.maven.UpgradePluginVersion:
-      groupId: org.springframework.boot
-      artifactId: spring-boot-maven-plugin
-      newVersion: @spring-boot-version@
-  - org.openrewrite.maven.UpgradeDependencyVersion:
-      groupId: org.springframework
-      artifactId: "*"
-      newVersion: @springframework-version@
-  - org.openrewrite.maven.UpgradeParentVersion:
-      groupId: org.springframework.boot
-      artifactId: spring-boot-starter-parent
-      newVersion: @spring-boot-version@
-  - org.openrewrite.maven.RemoveDependency:
-       groupId: org.apache.camel.springboot
-       artifactId: camel-k-starter
-  - org.openrewrite.maven.RemoveDependency:
-        groupId: org.apache.camel.springboot
-        artifactId: camel-etcd3-starter
\ No newline at end of file
+  - org.apache.camel.upgrade.camel413.CamelSpringBootMigrationRecipe
```

Comment:
mhhh I'd like to keep the latest as is, I would not reference older SpringBoot recipes, this way is much more clear, moreover, the latest.yaml is special, it contains additional stuff that are not present in other recipes.